test/types/string/parIter/cyclicUTF8.chpl

```diff
@@ -0,0 +1,14 @@
+use CyclicDist;
+
+config const n = 10;
+
+var s= "çş"*n;
```

Comment:
I think I've convinced myself that this is correct / "proves" that following a cyclic leader is working correctly, but given the number of locales == the number of codepoints in this repeating sequence, it'd be more obvious if this were a string that didn't have repetitions in it (where it wouldn't even have to all be fancy UTF-8 characters as long as some of them were, I think).